libs/sdk-ui-kit/src/Header/HeaderInviteButton.tsx

```diff
@@ -0,0 +1,16 @@
+// (C) 2022 GoodData Corporation
```

Comment:
Please rename the component, as it's not a button, is it?

`HeaderInviteButton.tsx` → `HeaderInvite.tsx`